lib/v2/nicesss/search.js

```diff
@@ -0,0 +1,25 @@
+const utils = require('./utils');
+
+module.exports = async (ctx) => {
+    const limit = ctx.params.limit || 20;
+    const items = [];
+    const keyword = decodeURIComponent(ctx.params.keyword);
+
+    let pages = Math.ceil(limit / 10);
+    pages = pages > 10 ? 10 : pages;
+    const urls = Array.from({ length: pages }, (_, i) => `https://www.nicesss.com/wp-json/wp/v2/posts?search=${keyword}&page=${i + 1}`);
+
+    const itemPromises = urls.map((url) => utils.processUrl(url, ctx));
+    const allItems = await Promise.all(itemPromises);
+    items.push(...allItems.flat());
```

Comment:
Please extract data from the first page only.

As mentioned before in https://github.com/DIYgod/RSSHub/pull/14364#pullrequestreview-1862510879, you can adjust the no. of articles returned in the API using `per_page` search parameter.